stock_customer_deposit/models/stock_warehouse.py

```diff
@@ -183,11 +183,15 @@ def _get_customer_deposit_picking_type_update_values(self):
         customer_loc, supplier_loc = self._get_partner_locations()
```

Comment:
this is dead code now ?

Reply:
Yes, you're right. I've removed dead code.